environment-model/environment-model_shacl.ttl

```diff
@@ -3,134 +3,286 @@
+            sh:path environment-model:y ] ;
+    sh:targetClass environment-model:Coordinate2D .
+    
+environment-model:ProjectLocationShape a sh:NodeShape ;
+    sh:property [ skos:example "nan" ;
+            sh:datatype xsd:string ;
+            sh:description "State of project area"@en ;
+            sh:message "Validation of state failed!"@en ;
+            sh:maxCount 1 ;
+            sh:name "state"@en ;
+            sh:order 1 ;
+            sh:path environment-model:state ],
+        [ skos:example "nan" ;
             sh:datatype xsd:string ;
             sh:description "nan"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:message "Validation of city failed!"@en ;
+            sh:maxCount 1 ;
+            sh:name "city"@en ;
+            sh:order 3 ;
+            sh:path environment-model:city ],
+        [ skos:example "nan" ;
+            sh:datatype xsd:string ;
+            sh:description "Region of project area"@en ;
+            sh:message "Validation of region failed!"@en ;
+            sh:maxCount 1 ;
+            sh:name "region"@en ;
+            sh:order 2 ;
+            sh:path environment-model:region ],
+        [ skos:example "Germany, GER" ;
+            sh:datatype xsd:string ;
+            sh:description "Country code as ISO 3166-1, alpha-2; "@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of country failed!"@en ;
+            sh:name "country"@en ;
+            sh:order 0 ;
+            sh:path environment-model:country ;
+            sh:pattern "^[a-zA-Z]{2}-(?:[a-zA-Z]{1,3}|[0-9]{1,3})$" ],
+        [ skos:example "Lat: 1.234- 1.345; Lon 45.23-45.34" ;
+            sh:description "Bounding box with lat/lon values in WGS84"@en ;
+            sh:message "Validation of boundingBox failed!"@en ;
+            sh:name "boundingBox"@en ;
+            sh:node environment-model:BoundingBoxShape ;
+            sh:minCount 1 ;
+            sh:maxCount 1 ;
+            sh:order 5 ;
+            sh:path environment-model:boundingBox ],
+        [ skos:example "Autobahn A8, Großraum Ingolstadt" ;
+            sh:datatype xsd:string ;
+            sh:description "Description of the location, dependen if it is a more trajectory style data or area style data"@en ;
+            sh:message "Validation of relationOrArea failed!"@en ;
+            sh:maxCount 1 ;
+            sh:name "relationOrArea"@en ;
+            sh:order 4 ;
+            sh:path environment-model:relationOrArea ] ;
+    sh:targetClass environment-model:ProjectLocation .     
+
+environment-model:FormatShape a sh:NodeShape ;
+    sh:property [ skos:example "Unreal DataSmith, Autodesk FBX, OpenSceneGraph" ;
+            sh:datatype xsd:string ;
+            sh:description "Format type definition"@en ;
+            sh:in ("Autodesk FBX" "GLTF" "OpenSceneGraph" "Unreal DataSmith") ;
+            sh:maxCount 1 ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
```

Comment:
Is "type" the right name here?

Reply:
below is "verison of data format". I think it refers to e.g. GLTF? - in this case "dataFormat" would make more sense then "type"?

Reply:
change to dataType in category format